charts/kubeai/charts/crds/crds/kubeai.org_models.yaml

```diff
@@ -964,6 +966,49 @@ spec:
                 items:
                   type: string
                 type: array
+              autoscaling:
+                description: |-
+                  Autoscaling configuration.
+                  Will be set from the AutoscalingProfile if not specified.
+                properties:
+                  disabled:
+                    description: Disabled is used to disable autoscaling.
+                    type: boolean
+                  maxReplicas:
+                    default: 3
+                    description: MinReplicas is the minimum number of Pod replicas
```

Comment:
description is incorrect. It mentions MinReplicas instead of maxReplicas

Reply:
probably need to regenerate this CRD?

Reply:
Fixed, good catch